         <html lang="en">
        <head>
        <meta charset="UTF-8">
        <title>Han Sun, PhD</title>
        <style>
        /* borrowed from mike, https://github.com/mikepqr/resume.md */

body {
    color: #000000;
    background: #EEEEEE;
    font: 1.1em "Times New Roman";
    line-height: 1.2;
    margin: 40px 0;
}
#resume {
    margin: 0 auto;
    max-width: 800px;
    padding: 40px 60px;
    background: #FFFFFF;
    border: 1px solid #CCCCCC;
    box-shadow: 2px 2px 4px #AAAAAA;
    -webkit-box-shadow: 2px 2px 4px #AAAAAA;
}
h1 {
    text-align: center;
    font-size: 200%;
    margin: 0;
    padding: 0;
    color: #0077B5;
}
h2 {
    border-bottom: 1px solid #000000;
    text-transform: uppercase;
    font-size: 130%;
    margin: 1em 0 0 0;
    padding: 0;
    color: #0077B5;
}
h3 {
    font-size: 100%;
    margin: 0.8em 0 0.3em 0;
    padding: 0;
    display: flex;
    justify-content: space-between;
}
p {
    margin: 0 0 1.5em 0;
    padding: 0;
    }
ul {
    padding: 0;
    margin: 0.8em 1.5em;
    }
ol {
    padding: 0;
    margin: 0.8em 1.5em;
    }
/* ul immediately after h1 = contact list */
h1 + ul {
    text-align: center;
    margin: 1.5em 1.5em;
    padding: 0;
    }
h1 + ul > li {
    display: inline;
    *white-space: pre;*/
    *list-style-type: circle;*/
}

h1 + ul > li:after {
    content: "    \2022    ";
}

h1 + ul > li:last-child:after {
    content: "";
}

/* p immediately after contact list = summary */
h1 + ul + p {
    margin: 1em 0;
}
@media print {
    body {
        font-size: 10pt;
        margin: 0;
        padding: 0;
        background: none;
    }
    #resume {
        margin: 0;
        padding: 0;
        border: 0px;
        background: none;
        box-shadow: none;
        -webkit-box-shadow: none;
    }
    /* Make links black in PDF */
    /* Move this outside the print block to apply this in HTML too */
    a, a:link, a:visited, a:hover {
        color: #000000;
        text-decoration: underline;
    }
}
@page {
    /* Change margins and paper size of PDF */
    /* https://developer.mozilla.org/en-US/docs/Web/CSS/@page */
    size: letter;
    margin: 0.5in 0.8in;
}
@media screen and (max-width: 800px) {
    body {
        font-size: 16pt;
        margin: 0;
        padding: 0;
        background: #FFFFFF !important;
    }
    #resume {
        margin: 0;
        padding: 1em;
        border: 0px;
        background: none;
        box-shadow: none;
        -webkit-box-shadow: none;
    }
}

        </style>
        </head>
        <body>
        <div id="resume">
        <h1>Han Sun, PhD</h1>
<ul>
<li><a href="&#109;&#97;&#105;&#108;&#116;&#111;&#58;&#104;&#97;&#110;&#115;&#117;&#110;&#64;&#115;&#116;&#97;&#110;&#102;&#111;&#114;&#100;&#46;&#101;&#100;&#117;">&#104;&#97;&#110;&#115;&#117;&#110;&#64;&#115;&#116;&#97;&#110;&#102;&#111;&#114;&#100;&#46;&#101;&#100;&#117;</a></li>
<li><a href="https://www.linkedin.com/in/sunhanice/">LinkedIn</a></li>
<li><a href="https://github.com/HaniceSun">GitHub</a></li>
<li>408-800-1190</li>
</ul>
<h2>Experience</h2>
<h3><span>Research Scientist/Senior Computational Biologist</span> <span>2020 &ndash; </span></h3>
<p>Department of Pediatrics, School of Medicine, Stanford University, <strong><a href="https://med.stanford.edu/genomics-of-diabetes.html">Prof. Anna Gloyn</a></strong></p>
<h3><span>Postdoc</span> <span>2015 &ndash; 2019</span></h3>
<p>Department of Genetics, School of Medicine, Stanford University, <strong><a href="https://web.stanford.edu/group/steinmetzlab/cgi-bin/wordpress/?page_id=59">Prof. Lars Steinmetz</a></strong></p>
<h2>Education</h2>
<h3><span>PhD, Bioinformatics</span> <span>2009 &ndash; 2014</span></h3>
<p>University of Chinese Academy of Sciences, <strong>Prof. Yixue Li</strong> and <strong>Prof. Lu Xie</strong></p>
<h3><span>Bachelor, Software Engineering</span> <span>2004 &ndash; 2008</span></h3>
<p>East China Normal University</p>
<h2>Skills</h2>
<ul>
<li>
<p>Python, R, C, git, conda, slurm, Linux, HPC, and cloud computing (AWS and GCP)</p>
</li>
<li>
<p>Containerization (Docker and Singularity) and workflow management (Snakemake and Nextflow)</p>
</li>
<li>
<p>Bioinformatics (blast, plink, bwa, samtools, STAR, featureCounts, GATK, seurat, scanpy)</p>
</li>
<li>
<p>Biostatistics (hypothesis testing, confidence interval, power calculation, bayesian inference, DESeq2, statsmodels)</p>
</li>
<li>
<p>Datasets (Ensembl, dbSNP, gnomAD, GTEx, COSMIC, TCGA, ENCODE, GEO, SRA, EGA, dbGaP, All of Us)</p>
</li>
<li>
<p>Multiomics: Genomics - whole genome sequencing (WGS), whole exome sequencing (WES), genotyping (genotyping array); Transcriptomics - bulk and single cell RNA-Seq (10x Genomics/Smart-Seq2), long reads sequencing (PacBio/Nanopore); Proteomics - Label-free and Labelled tandem mass spectrometry (MS/MS); Epigenomics - bulk and single nucleus ATAC-Seq (chromatin accessibility), whole genome bisulfite sequencing and methylation array (methylation status), ChIP-Seq/CUT&amp;RUN (transcription factor binding)</p>
</li>
<li>
<p>Linkage analysis, quantitative trait locus mapping (QTL), genome-wide association studies (GWAS), imputation, permutation, fine-mapping, and colocalization</p>
</li>
<li>
<p>Machine Learning (scikit-learn: linear model, generalized linear model, Random Forrest; dataset splitting, cross validation, overfitting) and Deep Learning (PyTorch and Tensorflow: CNN, ResNet, Transformer, GPT, etc.)</p>
</li>
</ul>
<h2>Projects</h2>
<h3>Splicing-mediated transcript fusion and oncogene activation in multiple cancers</h3>
<ul>
<li>
<p>Designed the project to explore splicing-mediated gene fusion events in cancer patients.</p>
</li>
<li>
<p>Reanalyzed both whole exome sequencing (WES) and RNA-Seq data from over 10,000 patients across 33 cancer types from the TCGA project using a negative binomial-based generalized linear model.</p>
</li>
<li>
<p>Discovered significant correlations between somatic mutations and the splicing of adjacent genes (transcript fusion), including the recurrent tumor-specific CDKN1A-RAB44 hit in bladder, stomach, and skin cancers.</p>
</li>
<li>
<p>Lead the collaboration with other labs to validate the finding using CRISPR/Cas9 editing, RT-PCR, Sanger sequencing, qPCR, RNA-Seq, and phenotyping assays.</p>
</li>
<li>
<p>Analyzed the data, interpreted the results, and prepared the manuscript. Proposed a splicing-mediated readthrough stabilization (SRS) model as a novel mechanism for oncogene activation.</p>
</li>
</ul>
<h3>RNA-binding protein RBM20-deficient dilated cardiomyopathy</h3>
<ul>
<li>
<p>Identified a suspicious missense mutation in RBM20 via linkage analysis in a family affected by dilated cardiomyopathy through analyzing whole exome sequencing (WES) and whole genome sequencing (WGS) data.</p>
</li>
<li>
<p>Collaborated with colleagues to validate the mutation using CRISPR/Cas9 editing in human induced pluripotent stem (iPS) cells and differentiating them into cardiomyocytes.</p>
</li>
<li>
<p>Identified splicing targets of RBM20 using RNA-Seq data from humans and pigs given that RBM20 is an RNA-binding protein.</p>
</li>
<li>
<p>By applying drug repositioning strategy in thousands of compounds from the Integrated Network-based Cellular Signatures (LINCS) project, we nominated retinoic acid and cardiac glycoside that may potentially rescue the phenotype.</p>
</li>
<li>
<p>By integrating gene expression data from thousands of tissues in the The Genotype-Tissue Expression (GTEx) project, I proposed a model for the tissue-specific splicing of a mitochondrial inner membrane protein, IMMT, suggesting a molecular link between energy supply deficiencies and dilated cardiomyopathy.</p>
</li>
</ul>
<h3>Mutation and regulatory elements underlying neonatal diabetes</h3>
<ul>
<li>
<p>Identified a suspicious intronic mutation in neonatal diabetes by analyzing whole genome sequencing (WGS) and array-based genotyping data from mice.</p>
</li>
<li>
<p>Collaborated with colleagues to successfully validate that the mutation is causing diabetes by introducing it into both B6 and NOD mice using CRISPR/Cas9 editing.</p>
</li>
<li>
<p>Proposed a mechanism where the mutation creates a novel splice site, resulting in a longer transcript but truncated protein, which has been verified as well by RT-PCR.</p>
</li>
<li>
<p>Working with bulk and single-cell data (scRNA-Seq and scATAC-Seq) throughout development to investigate potential extra regulatory elements and understand why the phenotype is specific to pancreas.</p>
</li>
<li>
<p>Working on large-scale QTL mapping (eQTL, caQTL, and pQTL) on multiomics datasets of human islets to study the genetic architecture of diabetes.</p>
</li>
</ul>
<h3>Applying machine learning/deep learning in omics data</h3>
<ul>
<li>
<p>Applied machine learning models (Random Forrest, XGBoost, SVM, etc.) to predict ancestry from genotyping data.</p>
</li>
<li>
<p>Developed deep learning models (CNN, ResNet, GPT, Llama2) in predicting selenocysteine and tissue-specific splicing from DNA sequences.</p>
</li>
</ul>
<h2>Certifications</h2>
<ul>
<li>
<p>Machine Learning Specialization, Andrew Ng, <a href="https://www.coursera.org/account/accomplishments/specialization/H9HDDLCTUTHS">Coursera</a></p>
</li>
<li>
<p>Deep Learning Specialization, Andrew Ng, <a href="https://www.coursera.org/account/accomplishments/specialization/DUM8L98A972Q">Coursera</a></p>
</li>
</ul>
<h2>Patents</h2>
<ul>
<li>Methods of Treatment, Genetic Screening, and Disease Models for Heart Conditions Associated with RBM20 Deficiency. Francesca Briganti, Lars M. Steinmetz, Han Sun, and Wu Wei. WO 2020/092171.</li>
</ul>
<h2>Publications</h2>
<ol>
<li>
<p><strong>Han Sun</strong>, Huiying Yan, Kathryn Bieging-Rolett, Michelle Nguyen, William F. Mueller, Zhuanfen Cheng, Hong Zeng, Laura Attardi, Wu Wei, and Lars M. Steinmetz et al. &ldquo;CDKN1A-RAB44 transcript fusion and oncogene activation in cancers.&rdquo;, <a href="https://www.biorxiv.org/content/biorxiv/early/2019/02/22/111856.full.pdf">bioRxiv</a>, <strong>co-first author</strong>.</p>
</li>
<li>
<p>Francesca Briganti, <strong>Han Sun</strong>, Wu Wei, Jingyan Wu, Chenchen Zhu, Martin Liss, Ioannis Karakikes et al. &ldquo;iPSC Modeling of RBM20-Deficient DCM Identifies Upregulation of RBM20 as a Therapeutic Strategy.&rdquo; <u>Cell Reports</u> 32, no. 10 (2020): 108117, <strong>co-first author</strong>.</p>
</li>
<li>
<p><strong>Han Sun</strong>, Seth A Sharp,  Nicole A J Krentz, Anna Gloyn et al., &ldquo;Deep learning prediction of selenocystine element and its application in diabetes&rdquo;, in preparation, <strong>co-first author</strong>.</p>
</li>
<li>
<p>Russ-Silsby, James, Yunkyeong Lee, Varsha Rajesh, <strong>Han Sun</strong>, Mahsa Amoli, Nasser Ali Mirhosseini, Tushar Godbole, Matthew B. Johnson et al. &ldquo;Complete Loss of PAX4 causes Transient Neonatal Diabetes in Humans.&rdquo; Molecular Metabolism (2025): 102201.</p>
</li>
<li>
<p>de Winter, Twan JJ, Miha Sovrovic, Esmee Dekker, Natascha de Graaf, <strong>Han Sun</strong>, James D. Johnson, Anna L. Gloyn, Francoise Carlotti, Eelco JP de Koning, and Anna Alemany. &ldquo;Cell type-specific eQTL detection from single-cell RNA-seq reveals post-transcriptional regulatory mechanisms in human islets.&rdquo; bioRxiv (2025): 2025-01.</p>
</li>
<li>
<p>Evans-Molina, Carmella, Yasminye D. Pettway, Diane C. Saunders, Seth A. Sharp, Thomas SR Bate, <strong>Han Sun</strong>, Heather Durai et al. &ldquo;Heterogeneous endocrine cell composition defines human islet functional phenotypes.&rdquo; bioRxiv (2024): 2024-11.</p>
</li>
<li>
<p>Kolic, Jelena, WenQing Grace Sun, Haoning Howard Cen, Jessica D. Ewald, Jason C. Rogalski, Shugo Sasaki, <strong>Han Sun</strong>, et al. &ldquo;Proteomic predictors of individualized nutrient-specific insulin secretion in health and disease.&rdquo; <u>Cell Metabolism</u> 36, no. 7 (2024): 1619-1633.</p>
</li>
<li>
<p>Ewald, Jessica D., Yao Lu, Cara E. Ellis, Jessica Worton, Jelena Kolic, <strong>Han Sun</strong>, Shugo Sasaki, Dahai Zhang et al. &ldquo;HumanIslets. com: Improving accessibility, integration, and usability of human research islet data.&rdquo; <u>Cell Metabolism</u> (2024).</p>
</li>
<li>
<p>Rottner, Antje K., Yingying Ye, Elena Navarro-Guerrero, Varsha Rajesh, Alina Pollner, <strong>Han Sun</strong>, Romina J. Bevacqua, Jing Yang et al. &ldquo;A genome-wide CRISPR screen identifies CALCOCO2 as a regulator of beta cell function influencing type 2 diabetes risk.&rdquo; <u>Nature Genetics</u> 55, no. 1 (2023): 54-65.</p>
</li>
<li>
<p>Lau, Hwee Hui, Nicole AJ Krentz, Fernando Abaitua, Marta Perez-Alcantara, <strong>Han Sun</strong>, Jun-Wei Chan, Jila Ajeian, Soumita Ghosh et al. &ldquo;PAX4 loss of function increases diabetes risk by altering human pancreatic endocrine cell development.&rdquo; <u>Nature Communications</u> 14, no. 1 (2023): 6119.</p>
</li>
<li>
<p>Mattis, Katia K., Nicole AJ Krentz, Christoph Metzendorf, Fernando Abaitua, Aliya F. Spigelman, <strong>Han Sun</strong>, Jennifer M. Ikle et al. &ldquo;Loss of RREB1 in pancreatic beta cells reduces cellular insulin content and affects endocrine cell gene expression.&rdquo; <u>Diabetologia</u> 66, no. 4 (2023): 674-694.</p>
</li>
<li>
<p>Cortez, Briana N., Hui Pan, Samuel Hinthorn, <strong>Han Sun</strong>, Nicola Neretti, Anna L. Gloyn, and Cristina Aguayo-Mazzucato. &ldquo;Heterogeneity of increased biological age in type 2 diabetes correlates with differential tissue DNA methylation, biological variables, and pharmacological treatments.&rdquo; <u>GeroScience</u> (2023): 1-21.</p>
</li>
<li>
<p>Torres, Jason M., <strong>Han Sun</strong>, Vibe Nylander, Damien J. Downes, Martijn van de Bunt, Mark I. McCarthy, Jim R. Hughes, and Anna L. Gloyn. &ldquo;Inferring causal genes at type 2 diabetes GWAS loci through chromosome interactions in islet cells.&rdquo; <u>Wellcome Open Research</u> 8 (2023).</p>
</li>
<li>
<p>Alghamdi, Tamadher A., Nicole AJ Krentz, Nancy Smith, Aliya F. Spigelman, Varsha Rajesh, Alokkumar Jha, <strong>Han Sun</strong>, Mourad Ferdaoussi et al. &ldquo;Zmiz1 is required for mature β-cell function and mass expansion upon high fat feeding.&rdquo; <u>Molecular Metabolism</u> 66 (2022): 101621.</p>
</li>
<li>
<p>Chenchen Zhu, Jingyan Wu, <strong>Han Sun</strong>, Francesca Briganti, Benjamin Meder, Wu Wei, and Lars M. Steinmetz. &ldquo;Single-molecule, full-length transcript isoform sequencing reveals disease-associated RNA isoforms in cardiomyocytes.&rdquo; <u>Nature Communications</u> 12, no. 1 (2021): 1-9.</p>
</li>
<li>
<p>Benedikt Rauscher, William F Mueller, Sandra Clauder-Münster, Petra Jakob, M Saiful Islam, <strong>Han Sun</strong>, Sonja Ghidelli-Disse, Markus Boesche, Marcus Bantscheff, Hannah Pflaumer, Paul Collier, Bettina Haase, Songjie Chen, Rene Hoffman, Guangwen Wang, Vladimir Benes, Gerard Drewes, Michael Snyder, Lars M Steinmetz. &ldquo;Patient-derived gene and protein expression signatures of NGLY1 deficiency&rdquo;, <u>Journal of Biochemistry</u>, 2021</p>
</li>
<li>
<p>Semih Calamak, Menekse Ermis, <strong>Han Sun</strong>, Saiful Islam, Michael Sikora, Michelle Nguyen, Vasif Hasirci, Lars M. Steinmetz, and Utkan Demirci. &ldquo;A Circulating Bioreactor Reprograms Cancer Cells Toward a More Mesenchymal Niche.&rdquo; <u>Advanced Biosystems</u> 4, no. 2 (2020): 1900139, <strong>co-first author</strong>.</p>
</li>
<li>
<p>Jay W Schneider, Saji Oommen, Muhammad Y Qureshi, Sean C Goetsch, Rhianna S Sundsbak, Wei Guo, Mingming Sun, <strong>Han Sun</strong>, Dennis A Webster, Alex W Coutts, Francesca Briganti, Wu Wei, Lars Steinmetz, Daniel F Carlson, and Timothy J. Nelson et al. &ldquo;Dysregulated ribonucleoprotein granules promote cardiomyopathy in RBM20 gene-edited pigs&rdquo;, <u>Nature Medicine</u> (2020): 1-13.</p>
</li>
<li>
<p>William F. Mueller, Petra Jakob, <strong>Han Sun</strong>, Sandra Clauder-Münster, Sonja Ghidelli-Disse, Diana Ordonez, Markus Boesche, Marcus Bantscheff, Paul Collier, Bettina Haase, Vladimir Benes, Malte Paulsen, Peter Sehr, Joe Lewis, Gerard Drewes, Lars M. Steinmetz. &ldquo;Loss of N-glycanase 1 Alters Transcriptional and Translational Regulation in K562 Cell Lines.&rdquo; <u>G3: Genes, Genomes, Genetics</u> (2020).</p>
</li>
<li>
<p>Arne H. Smits, Frederik Ziebell, Gerard Joberty Nico Zinn, William F. Mueller, Sandra Clauder-Munster, Paola Grandi, Petra Jakob, Anne-Marie Michon, <strong>Hanice Sun</strong>, Karen Tessmer, Tilmann Burckstummer, Marcus Bantscheff, Lars M. Steinmetz, Gerard Drewes, and Wolfgang Huber. &ldquo;Biological plasticity rescues target activity in CRISPR knock outs.&rdquo;, <u>Nature Methods</u> 2019 Oct 28:1-7.</p>
</li>
<li>
<p>Suo, Lun, Yu Xiao Zhou, Li Ling Jia, Hai Bo Wu, Jin Zheng, Qi Feng Lyu, Li Hua Sun, <strong>Han Sun</strong>, and Yan Ping Kuang. &ldquo;Transcriptome profiling of human oocytes experiencing recurrent total fertilization failure.&rdquo; <u>Scientific Reports</u> 8, no. 1 (2018): 17890.</p>
</li>
<li>
<p><strong>Han Sun</strong>, Chen Chen, Baofeng Lian, Menghuan Zhang, Xiaojing Wang, Bing Zhang, Yixue Li, Pengyuan Yang, and Lu Xie. 2015. &ldquo;Identification of HPV Integration and Gene Mutation in HeLa Cell Line by Integrated Analysis of RNA-Seq and MS/MS Data.&rdquo; <u>Journal of Proteome Research</u> 14 (4): 1678–86, <strong>co-first author</strong>.</p>
</li>
<li>
<p><strong>Han Sun</strong>, Chen Chen, Meng Shi, Dandan Wang, Mingwei Liu, Daixi Li, Pengyuan Yang, Yixue Li, and Lu Xie. 2014. &ldquo;Integration of Mass Spectrometry and RNA-Seq Data to Confirm Human Ab Initio Predicted Genes and lncRNAs.&rdquo; <u>Proteomics</u> 14 (23-24): 2760–68, <strong>co-first author</strong>.</p>
</li>
<li>
<p><strong>Han Sun</strong>, Xiaobin Xing, Jing Li, Fengli Zhou, Yunqin Chen, Ying He, Wei Li, et al. 2013. &ldquo;Identification of Gene Fusions from Human Lung Cancer Mass Spectrometry Data.&rdquo; <u>BMC Genomics</u> 14 Suppl 8 (December): S5, <strong>co-first author</strong>.</p>
</li>
<li>
<p>Zhen-Hao Liu, Bao-Feng Lian, Qiong-Zhu Dong, <strong>Han Sun</strong>, Jin-Wang Wei, Yuan-Yuan Sheng, Wei Li, et al. 2018. &ldquo;Whole-Exome Mutational and Transcriptional Landscapes of Combined Hepatocellular Cholangiocarcinoma and Intrahepatic Cholangiocarcinoma Reveal Molecular Diversity.&rdquo; <u>Biochimica et Biophysica Acta</u> 1864 (6 Pt B): 2360–68.</p>
</li>
<li>
<p>Menghuan Zhang, Hong Li, Ying He, <strong>Han Sun</strong>, Li Xia, Lishun Wang, Bo Sun, et al. 2015. &ldquo;Construction and Deciphering of Human Phosphorylation-Mediated Signaling Transduction Networks.&rdquo; <u>Journal of Proteome Research</u> 14 (7): 2745–57.</p>
</li>
<li>
<p>Wei Li, Jian Yu, Baofeng Lian, <strong>Han Sun</strong>, Jing Li, Menghuan Zhang, Ling Li, Yixue Li, Qian Liu, and Lu Xie. 2015. &ldquo;Identifying Prognostic Features by Bottom-up Approach and Correlating to Drug Repositioning.&rdquo; <u>PloS One</u> 10 (3): e0118672.</p>
</li>
<li>
<p>Jing Li, Jia Jia, Hong Li, Jian Yu, <strong>Han Sun</strong>, Ying He, Daqing Lv, et al. 2014. &ldquo;SysPTM 2.0: An Updated Systematic Resource for Post-Translational Modification.&rdquo; <u>Database: The Journal of Biological Databases and Curation</u> 2014 (April): bau025.</p>
</li>
<li>
<p>Yulin Dai, Shengdi Li, Xiao Dong, <strong>Han Sun</strong>, Chao Li, Zhi Liu, Beili Ying, Guohui Ding, and Yixue Li. 2013. &ldquo;The de Novo Sequence Origin of Two Long Non-Coding Genes from an Inter-Genic Region.&rdquo; <u>BMC Genomics</u> 14 (Suppl 8): S6.</p>
</li>
<li>
<p>Li Xia, Tong-Dan Wang, Shao-Ming Shen, Meng Zhao, <strong>Han Sun</strong>, Ying He, Lu Xie, et al. 2013. &ldquo;Phosphoproteomics Study on the Activated PKCδ-Induced Cell Death.&rdquo; <u>Journal of Proteome Research</u> 12 (10): 4280–4301.</p>
</li>
<li>
<p>Jian Yu, Xiaobin Xing, Lingyao Zeng, Jiehuan Sun, Wei Li, <strong>Han Sun</strong>, Ying He, et al. 2012. &ldquo;SyStemCell: A Database Populated with Multiple Levels of Experimental Data from Stem Cell Differentiation Research.&rdquo; <u>PloS One</u> 7 (7): e35230.</p>
</li>
<li>
<p>Ying He, Menghuan Zhang, Yuanhu Ju, Zhonghao Yu, Daqing Lv, <strong>Han Sun</strong>, Weilan Yuan, et al. 2011. &ldquo;dbDEPC 2.0: Updated Database of Differentially Expressed Proteins in Human Cancers.&rdquo; <u>Nucleic Acids Research</u> 40 (D1): D964–71.</p>
</li>
<li>
<p>Xiao Chang, Yun Li, Jie Ping, Xiao-Bin Xing, <strong>Han Sun</strong>, Peng Jia, Chuan Wang, Yuan-Yuan Li, and Yi-Xue Li. 2011. &ldquo;EcoBrowser: A Web-Based Tool for Visualizing Transcriptome Data of Escherichia Coli.&rdquo; <u>BMC Research Notes</u> 4 (October): 405.</p>
</li>
<li>
<p>Xiao-Bin Xing, Qing-Run Li, <strong>Han Sun</strong>, Xing Fu, Fei Zhan, Xiu Huang, Jing Li, et al. 2011. &ldquo;The Discovery of Novel Protein-Coding Features in Mouse Genome Based on Mass Spectrometry Data.&rdquo; <u>Genomics</u> 98 (5): 343–51.</p>
</li>
</ol>         </div>
        </body>
        </html>
        
